### ISSUE TITLE
How to install for Intellij's ideavim

### ISSUE THREAD

**Issue by chientranthien** [wellle/targets.vim#284]
Hi,

May I know if there is any way to install for Intellij's ideavim?

Thanks

**Comment by edwardsavin** [comment]
I don't think that it's possible. If it is I would also like to know how because this is a great vim plugin.
